letters/letters.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+from redbot.core import commands
+
+
+class Letters(commands.Cog):
+    """Letters cog"""
+
+    def __init__(self, bot):
+        self.bot = bot
```

Comment:
We don't need this function because `self.bot` is never read. Can also remove bot from [here](https://github.com/rHomelab/LabBot-Cogs/blob/feat/letters/letters/__init__.py#:~:text=bot.add_cog(Letters(-,bot,-)))

Reply:
Resolved in 92ead07.